packages/components/image.css

```diff
@@ -0,0 +1,8 @@
+.img-responsive {
+    width: 100%;
+    height: 100%;
```

Comment:
ig, for height it will be auto. check once

Reply:
you are right deekshithmd.